ui/pages/token-allowance/token-allowance.stories-to-do.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import TokenAllowance from './token-allowance';
 
+// eslint-disable-next-line import/no-anonymous-default-export
```

Comment:
I don't think we need this one either
```suggestion
```

Reply:
This one is needed to address the lint error. Since it's a broken story and we changed it from .stories.js to .stories-to-do.js. We will remove it once this story will be fixed